gulpfile.js

```diff
@@ -25,8 +27,8 @@ gulp.task('js', function() {
 
 gulp.task('watch', function() {
 	gulp.watch('src/**/*.html', ['html']);
-	gulp.watch('src/**/*.css', ['css']);
+	gulp.watch('src/style.scss', ['sass']);
```

Comment:
Lets watch all *.scss files, so that the entire CSS compilations is automatic. In that way we do not have to add new `@import` every time we add a new view.